rust/cubestore/cubestore/src/queryplanner/info_schema/system_tables.rs

```diff
@@ -144,40 +144,10 @@ impl InfoSchemaTableDef for SystemTablesTableDef {
                 Box::new(|tables| {
                     let aggregates = tables
                         .iter()
-                        .map(|row| {
-                            row.table
-                                .get_row()
-                                .aggregate_columns()
-                                .as_ref()
-                                .map(|v| format!("{:?}", v))
-                        })
-                        .collect::<Vec<_>>();
-                    Arc::new(StringArray::from(
-                        aggregates
```

Comment:
Nothing to add here other than I'm very happy to see a bunch of code deleted from a long method :)